src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -158,15 +158,15 @@ impl HummockVersion {
     }
 
     /// This function does NOT dedup.
-    pub fn get_object_ids(&self) -> Vec<u64> {
+    pub fn get_object_ids(&self) -> HashSet<HummockSstableObjectId> {
```

Comment:
Change the comment above?

Reply:
Also, we can also remove the HashSet [here](https://github.com/risingwavelabs/risingwave/blob/7bc3c9268e89af4d0a3c0768dedcadef8e37246b/src/storage/backup/src/lib.rs#L72).